browser/react/components/Login.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { login, logout } from '../../redux/reducers/auth';
+
+/* ----------------- COMPONENT ------------------ */
+
+class Login extends React.Component {
+  constructor (props) {
```

Comment:
all of my following comments are basically trying to make this a dumb component.  I like dumb components.  I think they look cleaner.  I'm a big redux fan personally.